src/index.js

```diff
@@ -8,4 +9,71 @@ const displayAllMovies = async () => {
   displayMovies(display);
 };
 
-window.addEventListener('load', displayAllMovies);
\ No newline at end of file
+window.addEventListener('load', displayAllMovies);
+
+const shows = await getMovies();
+const arr = Array.from(shows);
+
+const parentElem = document.getElementById('liContainer');
+parentElem.addEventListener('click', (event) => {
+  const matcher = event.target.matches('.btn');
+  const eventElem = event.target;
+  const eventId = eventElem.id - 1;
+
+  if (matcher) {
+    const newDiv = document.createElement('div');
+    newDiv.className = 'mainContainer';
+    newDiv.innerHTML = `
+    <div class="mainContainer1">
+
+    <div class="closeBtd flex">
+        <img id="closeImg" src="./images/close_icon.png" alt="main image">
+    </div>
```

Comment:
It would be better to add some margin to the close button 👎 
<img width="364" alt="Capture1" src="https://github.com/Zohra-Neda/JavaScript-Capstone-Project/assets/58807251/fc95b764-0eb8-4743-b09f-299ece99ea30">

Reply:
resolved